website/blog/2024-04-10-roadmap-q2.md

```diff
@@ -0,0 +1,88 @@
+  name: Sam Richards
+  title: MACI team lead
+  url: https://x.com/samonchain
+  image_url: https://avatars.githubusercontent.com/u/8097623?v=4
+tags: [roadmap, rpgf, qf]
+---
+
+Greetings anons,
+
+Happy April! Glad to have you hear. A few updates to share with you all.
+
+Before we hop into our plans, let's take a look at Q1
+
+## Q1 in review
+
+In Q1 we aimed at [a few major goals](https://github.com/privacy-scaling-explorations/maci/discussions/859#discussioncomment-7849385) that all tie together:
+
+### ✅ Release MACI v1.2
+
+We completed a significant refactor to simplify the codebase and improve developer experience. After an internal audit, revamped documentation & educational resources, we released a new MACI version the world - see our [MACI v1.2 release post](/blog/maci-v1-2-0-release) to read up on all the groovy benefits of that upgrade.
+
+### ✅ clr.fund integration
+
+[clr.fund](https://clr.fund/#/), a quadratic funding (QF) project which has distributed millions of dollars to Ethereum public goods, is the longest-running production application built on MACI (since 2020!). Thanks primarily to the hard work of [yuetloo](https://github.com/yuetloo), clr.fund successfully upgraded from 0.x to our fresh v1.2 release.
+
+### ✅ ETH Latam QF round
+
+What better way to celebrate the annual [Ethereum conference of Latam](https://ethlatam.org/) than with a QF round? We teamed up with ETH Latam to support their efforts to run a successful QF round on clr.fund with this latest version of MACI. Over 60 attendees participated (using Zupass tickets as the round gatekeeper) to distribute over $30,000 DAI on Optimism to projects across Central America and the Carribean - [view the results here](https://qf.ethlatam.org/#/leaderboards/0x86F33909474c0dEf2Cb7F93d2eE0B8aF26112BF6/networks/optimism)!
+
+### ✅ ETH Colombia QF round
+
+Bonus points! Kudos also to the ETHColombia team who ran a [QF round for their community](https://www.ethcolombia.org/quadratic-funding-ethco-2024-q1) on the #RoadToETHLatam - [view the results here](https://qf.ethcolombia.org/#/leaderboards/0xa73Ec044b47186646D84D614b8a194dA3bE00260/networks/optimism)!
+
+## Q2 Roadmap
+
+Now let's look ahead at what we're excited to build over the next few months:
+
+### 🎯 MACI-RPGF
+
+RPGF has taken the public goods funding space by storm
+Building momentum ove the past years and stormed into the Zeitgeist after Optimisms monster >$100,000,000 round late last year.
+In February we [proposed MACI as a solution for Optimism's RPGF tech stack](https://gov.optimism.io/t/building-a-private-on-chain-implementation-for-retropgf/7733), thanks to its strong guarantees of correct execution, cencorship resistance, privacy, collusion resistance.
+
+Since we released a [proof-of-concept MACI-RPGF implementation](https://github.com/privacy-scaling-explorations/maci-rpgf/), we've received lots of positive feedback from the community and interest from communities in running RPGF rounds on MACI.
```

Comment:
nit: ... from the "community" and interest from "communities" --> maybe we could combine them?